### PR TITLE
Fixed several timezone issues

### LINKED ISSUE #50: Events show up on wrong timeslot because of timezone
I already added this as a comment to another bug but I assumed it would be more handy to just create a new bug ticket for it.

I'm encountering a similar problem for events as @amytencza in https://github.com/derekantrican/GAS-ICS-Sync/issues/34.

Events that have been created in a different time zone end up at the wrong time slot in the google calendar. In my case the Event is created in India and should apprear at 11:00 (which it does in the original calendar) but in google calendar it appears at 14:30.

When I check the logs I see the following:
The event has a date of **05:30:00 GMT-0700** but India should be at **GMT-5.5**. I tried to look into the code myself but I have the feeling my programming skills are a bit to limited to really be of support here.

I can do some additional debugging if required
Screenshot 2019-08-21 at 09 51 22

### PR DESCRIPTION
**Pull request by jonas0b1011001**
Fix #50, #34, #29

### CODE REVIEW

**Comment by derekantrican** [comment]
@jonas0b1011001 I assume you've tested this with a variety of icals? If so, looks good to me and I'll approve it (and probably even make a new update)

**Comment by jonas0b1011001** [comment]
I did, all good except this [one](http://ical.8to18.com/link/jacksonville/2018-2019/BBB%20V/EDT) from #23..still not sure were these events are actually supposed to be..

But feel free to test yourself, i'm sure i have not tested all possibilities.

File: Code.gs [hunk 1]
```diff
@@ -225,7 +223,11 @@ function main(){
 
           if(e.getStartTime().getTime() != fe.startTime.getTime() ||
              e.getEndTime().getTime() != fe.endTime.getTime())
-            e.setTime(fe.startTime, fe.endTime)
+            if (fe.isAllDay){
```
Comment (derekantrican) [inline Code.gs]: I assume all the 

```
  if (fe.isAllDay){
    e.setAllDayDates(fe.startTime, fe.endTime);
  }else{
    e.setTime(fe.startTime, fe.endTime);
  }
```

is supposed to be within the parent if statement? I'm not super fluent in JS, but do we need brackets around it then? If not, maybe just add them for clarity (also did some minor corrections to conform with conventions in the rest of the program):

```
if(e.getStartTime().getTime() != fe.startTime.getTime() ||
   e.getEndTime().getTime() != fe.endTime.getTime()){
  if (fe.isAllDay)
    e.setAllDayDates(fe.startTime, fe.endTime);
  else
    e.setTime(fe.startTime, fe.endTime);
}
```

Comment (jonas0b1011001) [reply Code.gs]: > is supposed to be within the parent if statement?
Correct.

Pretty sure brackets are not needed but adding them for clarity is always a good idea

Comment (derekantrican) [reply Code.gs]: Right. I'm writing up a "coding conventions" document for the wiki right now. Go ahead and push another commit with the suggested change and I'll approve it

**Pull request merged**